winui/Time-Picker/Getting-Started.md

```diff
@@ -108,7 +108,7 @@ sfTimePicker.SelectedTime = new DateTimeOffset(new DateTime(2021, 10, 29, 10, 45
 {% endhighlight %}
 {% endtabs %}
 
-![programmatic-time-selection-in-winui-time-picker](Getting-Started_images/programmatic-time-selection-in-winui-time-picker.png)
```

Comment:
Existing image names are fine. So, revert your image name changes in all places.

Reply:
changed